src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/UpdateRecallControllerTest.kt

```diff
@@ -168,4 +168,31 @@ class UpdateRecallControllerTest {
 
     assertThat(response, equalTo(ResponseEntity.ok(updatedRecall.toResponse())))
   }
+
+  @Test
+  fun `can update recall with lastReleaseDateTime and lastReleasePrison without changing other properties`() {
+    val recallId = ::RecallId.random()
+    val priorRecall = Recall(recallId, nomsNumber, recallLength = FOURTEEN_DAYS, agreeWithRecallRecommendation = false)
+    every { recallRepository.getByRecallId(recallId) } returns priorRecall
+    val lastReleaseDateTime = ZonedDateTime.now()
+    val updatedRecall = priorRecall.copy(recallType = FIXED, lastReleasePrison = "prison", lastReleaseDateTime = lastReleaseDateTime)
+    every { recallRepository.save(updatedRecall) } returns updatedRecall
+
+    val response = underTest.updateRecall(recallId, UpdateRecallRequest(lastReleasePrison = "prison", lastReleaseDateTime = lastReleaseDateTime))
```

Comment:
For future: I would favour two changes here:
* extracting the value "prison" to a val and using that 3 times, rather than the literal 3 times
* superseding that fixed value ("prison") with variable data - some form of random string.
But they can come after.